nym-wallet/src-tauri/src/wallet_storage/mod.rs

```diff
@@ -152,6 +153,38 @@ pub(crate) fn store_login_with_multiple_accounts(
     store_login_with_multiple_accounts_at_file(&filepath, mnemonic, hd_path, id, password)
 }
 
+/// Update a login with multiple accounts support
+pub(crate) fn update_encrypted_login(
```

Comment:
IMHO I think we should just reencrypt all logins, if nothing else just to keep the invariant that all logins have the same password